### ISSUE TITLE
`npx @roxi/routify init` errors due to undefined path argument

### ISSUE THREAD

**Issue by Lucretius** [roxiness/routify#325]
**bug description**
Running the first step to generate a project from scratch (`npx @roxi/routify init`) outputs the following error:
`The "path" argument must be of type string. Received type undefined`

I tried v2.10.5 with the same issue but running with 2.9.0 everything worked.

`npx @roxi/routify@v2.9.0 init` output a Routify project.

Something appears to have been introduced in the 2.10+ release which changed the behavior of the init command.

**Comment by jakobrosenberg** [comment]
Running `npx @roxi/routify@v2.10.5 init` I'm unable to reproduce the issue.
```
$: npx @roxi/routify@v2.10.5 init
[Routify] Fetching template
> cloned roxiness/routify-starter#master
[Routify] Installing dependencies
[Routify] Run "npm run dev" to start the server.
```

Commanderjs was recently upgraded and is most likely the cause of your error.  Would you be able to run the command in a debug terminal to shed further light on this?

**Comment by TomaszSzczucinski** [comment]
Hello,
I have the same issue on Linux Ubuntu. Node, npm and npx are updated. Here's output from npx:

```Debugger listening on ws://127.0.0.1:9229/ec2f873f-770f-485a-9ab0-91e7d8eef334
For help, see: https://nodejs.org/en/docs/inspector
internal/validators.js:125
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);
    ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at resolve (path.js:1080:7)
    at getParentModuleDir (/home/tom/.npm/_npx/501047/lib/node_modules/@roxi/routify/node_modules/configent/configent.js:155:24)
    at configent (/home/tom/.npm/_npx/501047/lib/node_modules/@roxi/routify/node_modules/configent/configent.js:38:29)
    at module.exports.input (/home/tom/.npm/_npx/501047/lib/node_modules/@roxi/routify/lib/utils/config.js:7:40)
    at Object.<anonymous> (/home/tom/.npm/_npx/501047/lib/node_modules/@roxi/routify/plugins/cli.js:5:48)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
```
2.9.0 version is working for me also without any problems.

PS. No problems on Windows 10.

**Comment by SarcevicAntonio** [comment]
Im running Ubuntu 20.04 in WSL2 Node version 10.19 and using `npx @roxi/routify init` just results in `The "path" argument must be of type string. Received type undefined`. 

I can confirm that `npx @roxi/routify@v2.9.0 init` works for me as well.

But then for me it gives me the following output when trying to `npm run dev` 

``` ~/routify  npm run dev

> svelte-app@1.0.0 dev /home/linkplay/routify
> run-p routify nollup


> svelte-app@1.0.0 routify /home/linkplay/routify
> routify


> svelte-app@1.0.0 nollup /home/linkplay/routify
> nollup -c

internal/validators.js:125
    throw new ERR_INVALID_ARG_TYPE(name, 'string', value);```

**Comment by jakobrosenberg** [comment]
I don't have WSL setup at the moment so if you can share any further details about the errors, that would be a great help. 🙂

**Comment by ghostdevv** [comment]
So in trying to reproduce this issue I did the following:

Test One (Success)
- Node 14.15.4
- WSL 2
- Ubuntu 20.04
- Command: npx @roxi/routify init

Test Two (Success)
- Node 12.20.1
- WSL 2
- Ubuntu 20.04
- Command: npx @roxi/routify init

Test Three (Failed) "The "path" argument must be of type string. Received type undefined"
- Node 10.23.2
- WSL 2
- Ubuntu 20.04
- Command: npx @roxi/routify init

Test Four (Failed) "The "path" argument must be of type string. Received type undefined"
- Node 8.17.0 
- WSL 2
- Ubuntu 20.04
- Command: npx @roxi/routify init

So it must be a issue with v10 - 8 and the latest version of routify so now I will test the version it was told it works on:

Test One (Success)
- Node 10.23.2
- WSL 2
- Ubuntu 20.04
- Command: npx @roxi/routify@2.9.0 init

Test One (Failed) "Unexpected token {"
- Node 8.17.0 
- WSL 2
- Ubuntu 20.04
- Command: npx @roxi/routify@2.9.0 init

So in conclusion:

@TomaszSzczucinski Your issue seems to be with configent which jake will probably get on, use npx @roxi/routify init and latest node version and you should have no issue

@Lucretius and @SarcevicAntonio I found the issue, for now update to node v12 or above and I will post back here when it's fixed

**Comment by ghostdevv** [comment]
If you want to track this issue: https://github.com/roxiness/configent/issues/1 (It's an issue with configent on the node versions you all are running)

So @TomaszSzczucinski @Lucretius @SarcevicAntonio if you don't want to update to 12.x or above you can track the issue here but I would recommend updating and that will solve the issue

I will close this when the configent issue is fixed and a new version of routify is pushed out

**Comment by jakobrosenberg** [comment]
This should be fixed in `v2.12.10`
